NGitLab.Mock/Config/GitLabProject.cs

```diff
@@ -0,0 +1,58 @@
+﻿using NGitLab.Models;
+
+namespace NGitLab.Mock.Config
+{
+    /// <summary>
+    /// Describe a GitLab project
+    /// </summary>
+    public class GitLabProject : GitLabObject<GitLabConfig>
+    {
+        public GitLabProject()
+        {
+            Commits = new GitLabCommitsCollection(this);
+            Issues = new GitLabIssuesCollection(this);
+            MergeRequests = new GitLabMergeRequestsCollection(this);
+            Permissions = new GitLabPermissionsCollection(this);
+            Labels = new GitLabLabelsCollection(this);
+            Milestones = new GitLabMilestonesCollection(this);
+        }
+
+        /// <summary>
+        /// Name (required)
+        /// </summary>
+        public string Name { get; set; }
+
+        public string Namespace { get; set; } = "functional";
+
+        public string DefaultBranch { get; set; } = "main";
```

Comment:
I think the server will set the default branch if not set. Do we need to set a default value here?